aries-backchannels/acapy/acapy_backchannel.py

```diff
@@ -100,6 +100,26 @@ def __init__(
             "presentation_acked": "done"
         }
 
+        # Aca-py : RFC
+        self.didExchangeResponderStateTranslationDict = {
+            "initial": "invitation-sent", # responder
+            "invitation": "invitation-received", # requester
+            "request": "request-received", # responder
+            "response": "response-sent", # responder
+            "?": "abandoned",
+            "active": "completed"
+        }
+
+        # Aca-py : RFC
+        self.didExchangeRequesterStateTranslationDict = {
+            "initial": "invitation-sent", # responder
+            "invitation": "invitation-received", # requester
+            "request": "request-sent", # requester
+            "response": "response-received", # requester
```

Comment:
Same as above.  In this case, the requester doesn't know when this happens.

Reply:
Copy/Paste error.